src/core/getExports.js

```diff
@@ -97,6 +97,14 @@ export default class ExportMap {
       return m // can't continue
     }
 
+    const docstyle = (context.settings && context.settings['import/docstyle']) || { jsdoc: true }
```

Comment:
Is there a better place to define plugin default `settings` so it gets merged correctly with any user config?